searchlib/src/tests/common/bitvector/bitvector_test.cpp

```diff
@@ -824,4 +825,23 @@ TEST(BitvectorTest, reset_term_field_match_data_on_unpack)
     EXPECT_TRUE(check_full_term_field_match_data_reset_on_unpack(true, true));
 }
 
+TEST(BitvectorTest, fixup_after_short_read)
```

Comment:
Consider adding a comment on (or renaming test method to indicate) what is fixed up (guard bit)